src/htmlGenerator.js

```diff
@@ -1,133 +1,137 @@
-﻿var HtmlGenerator = function() {
-  this.generate = function (node, siteTitle, title, navigationObject, navLevel) {
-    var doc = docType() + tag('html', head(title, navLevel) + body(node, siteTitle, navigationObject));
-    // There must be a better way to convert the character set?
-    var dashRegex = new RegExp(String.fromCharCode(8211), 'g');
-    var spaceRegex = new RegExp(String.fromCharCode(160), 'g');
-    return doc.replace(dashRegex, '-').replace(spaceRegex, ' ');
-  };
+﻿define([
+  'src/utilities'
+], function(Util) {

```

Comment:
I think in general people capitalize variables names for classes only, not objects (`var x = new X()`), but Util is just an object.